pkg/influx/recorder.go

```diff
@@ -93,7 +95,11 @@ func (r prometheusRecorder) measureConversionDuration(duration time.Duration) {
 	r.conversionDuration.WithLabelValues().Observe(duration.Seconds())
 }
 
-func (r prometheusRecorder) RegisterVersionBuildTimestamp() error {
+func (r prometheusRecorder) RegisterVersionBuildTimestamp(logger log.Logger) error {
+	if DockerTag == "unset" || CommitUnixTimestamp == "0" {
+		_ = level.Warn(logger).Log("influx.DockerTag and influx.CommitUnixTimestamp are unset")
```

Comment:
what does it mean if these are unset? 99% of the time nobody pays attentions to warnings

Reply:
In terms of impact to us, the argo workflow will wait forever waiting for the metric with the correct DockerTag to show up.

I added a print statement here when I was troubleshooting the problem, and it seemed like it might be helpful in the future so I turned it into a warning.

Reply:
if it being unset would mean the service can't start up/ hang forever, might as well just crash (log.fatal) rather than force someone to figure out why it's hung.

UNLESS, there are situations where things can be start up without the timestamp (someone running this OSS, tests), in which case we don't want to enforce this method of startup

Reply:
Oop sorry - the Argo workflow will hang forever, but the influx2cortex service itself is actually perfectly fine. Someone running this OSS and tests are indeed two great examples of why we wouldn't want want to crash in that case here.